### PR TITLE
RDCC-1567 - HMCTS Gradle Java plugin changes

### PR DESCRIPTION
**Pull request by pksaxena01**
### JIRA link (if applicable) ###

RDCC-1567

### Change description ###

Code changes related to HMCTS Checkstyle plugin for User Profile

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java [hunk 1]
```diff
@@ -41,12 +41,25 @@
...
+    private static final String ACCESS_TOKEN = "eyJ0eXAiOiJKV1QiLCJ6aXAiOiJOT05FIiwia2lkIjoiRm8rQXAybThDT3ROb290ZjF"
```
Comment (abhijitdiwan) [inline src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java:44]: please use concat rather +

Comment (pksaxena01) [reply src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java:44]: I have used concat() in production code, however i will update this class with concat()

File: src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileUpdator.java [hunk 2]
```diff
@@ -148,7 +150,8 @@ private HttpStatus getHttpStatusFromFeignException(FeignException ex) {
     private UserProfile validateUserStatus(String userId) {
         Optional<UserProfile> userProfileOptional = userProfileRepository.findByIdamId(userId);
         if (!userProfileOptional.isPresent()) {
-            throw new ResourceNotFoundException("could not find user profile for userId: or status is not active " + userId);
+            throw new ResourceNotFoundException("could not find user profile for userId: or status is not active "
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/UserProfileUpdator.java:153]: only issue I can see here if userId value null we can get null pointer...hence are we sure we are getting userId in all cases? with + null pointer we can't get. but with concat we my get null pointer.

File: src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java [hunk 3]
```diff
@@ -60,20 +61,25 @@ public void validateUserIsPresent(Optional<UserProfile> userProfile) {
         }
     }
 
-    public boolean validateUpdateUserProfileRequestValid(UpdateUserProfileData updateUserProfileData, String userId, ResponseSource source) {
+    public boolean validateUpdateUserProfileRequestValid(UpdateUserProfileData updateUserProfileData, String userId,
+                                                         ResponseSource source) {
         if (!validateUserProfileStatus(updateUserProfileData)) {
             auditService.persistAudit(BAD_REQUEST, source);
-            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has invalid status %s for userId: %s", updateUserProfileData.getIdamStatus(), userId);
+            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has "
```
Comment (abhijitdiwan) [inline src/main/java/uk/gov/hmcts/reform/userprofileapi/service/impl/ValidationHelperServiceImpl.java:68]: I think this is wrong no need to update concat in format. It fails keep as is  

            final String exceptionMsg = String.format("RequiredFieldMissingException - Update user profile request has invalid status %s for userId: %s", updateUserProfileData.getIdamStatus(), userId);

File: src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java [hunk 4]
```diff
@@ -41,12 +41,26 @@
...
+            .concat("4TWg0bGc3MFlBPSIsImFsZyI6IlJTMjU2In0.eyJzdWIiOiJmcmVnLXRlc3QtdXNlci1ZOHlqVURSeWpyQGZlZW1haWwuY29")
+            .concat("tIiwiYXV0aF9sZXZlbCI6MCwiYXVkaXRUcmFja2luZ0lkIjoiYTU2MDliYjYtYzEzYi00MjQ0LTg3ODItNDNmZGViMDZlMDB")
+            .concat("jIiwiaXNzIjoiaHR0cHM6Ly9mb3JnZXJvY2stYW0uc2VydmljZS5jb3JlLWNvbXB1dGUtaWRhbS1hYXQuaW50ZXJuYWw6ODQ")
+            .concat("0My9vcGVuYW0vb2F1dGgyL2htY3RzIiwidG9rZW5OYW1lIjoiYWNjZXNzX3Rva2VuIiwidG9rZW5fdHlwZSI6IkJlYXJlciI")
+            .concat("sImF1dGhHcmFudElkIjoiYWNjNmUyYTAtMWExYi00OGM3LWJmZGItNzI1NjllM2E1NjkzIiwiYXVkIjoicmQtcHJvZmVzc2l")
+            .concat("vbmFsLWFwaSIsIm5iZiI6MTU2OTQ0MTkxMSwiZ3JhbnRfdHlwZSI6ImF1dGhvcml6YXRpb25fY29kZSIsInNjb3BlIjpbIm9")
+            .concat("wZW5pZCIsInByb2ZpbGUiLCJyb2xlcyIsImNyZWF0ZS11c2VyIiwibWFuYWdlLXVzZXIiXSwiYXV0aF90aW1lIjoxNTY5NDQ")
+            .concat("xOTExMDAwLCJyZWFsbSI6Ii9obWN0cyIsImV4cCI6MTU2OTQ1NjMxMSwiaWF0IjoxNTY5NDQxOTExLCJleHBpcmVzX2luIjo")
+            .concat("xNDQwMCwianRpIjoiY2Q5MWM0NjQtMzU0Zi00N2I2LTkwYTUtNWY2Y2U3NGUwYTY5In0.aLobAYYCxkmryzKV1stmag63h-n")
```
Comment (KOTLAPRA23) [inline src/contractTest/java/uk/gov/hmcts/reform/userprofileapi/IdamConsumerTest.java:53]: remove all the lines and just add only small sting . we are mocking so no need entire bearer token

**Pull request merged**